### PR TITLE
Civil disorder and DIAS draw proposals

### PR DESCRIPTION
**Pull request by johnpooch**
Implements two related Diplomacy mechanics:

**Civil disorder (#166)** — A member enters `civil_disorder` after two consecutive movement-phase NMRs. Their phase states are auto-confirmed at phase creation so they no longer block resolution. A game flips to `ABANDONED` when every active member is in CD. CD is permanent and is not reset by `revert_to_this_phase` (kept simple). Extensions still just delay the deadline — they do not absorb an NMR.

**DIAS draw proposals (#167)** — Proposers no longer pick a coalition; all active non-CD members are auto-included. CD members auto-accept and are excluded from victory. The SC threshold check is gone. Voting is now secret: the API no longer returns per-member vote rows; clients see only tally counts (accepted / rejected / pending) and the current user's own vote.

## Notable changes

- New `Member.civil_disorder` boolean (migration `0005`)
- New `PhaseManager._check_civil_disorder` runs after movement-phase adjudication, sends a notification, sets the flag via `bulk_update`
- `create_from_adjudication_data` now sets `orders_confirmed=member.civil_disorder` on new `PhaseState` rows
- `_check_abandonment` rewritten — game is abandoned iff every active member is in CD (was: "two consecutive movement phases with zero total orders")
- `DrawProposal.create_proposal` no longer takes `included_member_ids`; auto-includes active non-CD members; auto-accepts proposer and CD; calls `process_acceptance()` at creation if status is already `ACCEPTED` (fixes a corner case where the proposer is the only non-CD active member)
- `DrawProposalSerializer` exposes `accepted_count`, `rejected_count`, `pending_count`, `total_votes`, `included_member_ids`, `my_vote`; drops `votes`, `combined_sc_count`, `victory_threshold`
- Frontend: new `CivilDisorderBadge` component; `PlayerInfoContent`, `OrdersScreen`, and the map renderer surface CD; `ProposeDrawScreen` collapses to a confirmation screen; `DrawProposalsScreen` renders a tally instead of per-voter icons

## Tests

Backend: 1244 passing. Includes a full-flow integration test (`test_player_enters_civil_disorder_after_two_movement_phase_nmrs`) that drives the API through two NMR movement phases, plus an abandonment test, a DIAS accept/reject pair, and a cross-cutting CD-then-DIAS test that catches the proposer-only-active-member edge case.

Frontend: 306 passing. New screen-level tests for `ProposeDrawScreen`, `DrawProposalsScreen`, `OrdersScreen`, `PlayerInfoContent`, and the `CivilDisorderBadge`.

## Side fix

`test_active_game_create_move_order_fleet_to_named_coast` previously relied on a player never submitting orders — under the new CD logic that player would enter civil disorder by the third movement phase, throwing off the test's expected phase progression. Updated the test to submit a Berlin Hold order in Spring 1901.

## Open follow-ups

- No release notes entry yet — happy to add one if you want
- I have not driven a live dev server through the new screens; the test layers cover data flow and render shape but not real interaction. Worth a manual smoke before merge.

<sub>Generated with Claude Code</sub>

### CODE REVIEW

File: packages/web/src/components/InteractiveMap/toRenderState.test.ts [hunk 1]
```diff
@@ -93,11 +93,43 @@ describe("toRenderState", () => {
...
+    const state = toRenderState(
+      variant,
+      {
+        ...emptyPhase,
+        units: [
+          {
+            type: "Army",
+            nation: nations[0],
+            province: province("lon"),
+            dislodged: false,
```
Comment (johnpooch) [inline packages/web/src/components/InteractiveMap/toRenderState.test.ts:111]: How does this test pass if this is false?

Comment (johnpooch) [reply packages/web/src/components/InteractiveMap/toRenderState.test.ts:111]: `civilDisorder` and `dislodged` are independent in `toRenderState` — civilDisorder is set per-nation from the `civilDisorderNations` arg (6th positional), regardless of dislodged state. The test passes `["England"]`, so the first unit (England) gets `civilDisorder: true` and the second (France) gets `civilDisorder: false`. Dislodged just happens to be false in this test setup.

Happy to add a `dislodged: true` variant if you want the independence to be explicit.

[~ Automated via Claude](https://github.com/intercom/claude-plugins/blob/main/plugins/base/docs/external-message-attribution.md)
